.github/workflows/manualRelease.yml

```diff
@@ -0,0 +1,36 @@
+name: manual release
```

Comment:
Not sure if this one or the automated release are going to work.  Lots of moving parts.  Will debug in place if unsuccessful.